app/views/spaces/show.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<title>Details of the space <%= @space.name %></title>
```

Comment:
the title tag is only valid in the head section and won't render anything in the html body, please use the h1 tag